tests/commands/DriveToWaypointCommandTest.cpp

```diff
@@ -0,0 +1,96 @@
+#include <catch2/catch.hpp>
+#include <cmath>
+#include <cstdio>
+
+#include "../../src/commands/CommandBase.h"
+#include "../../src/commands/DriveToWaypointCommand.h"
+
+using namespace robot::types;
+using namespace std::chrono_literals;
+
+void assertApprox(double expectedThetaVel, double expectedXVel,
+                  const commands::command_t& actual, double angle = 1e-5,
+                  double vel = 1e-5) {
+  bool velEqual = abs(expectedThetaVel - actual.thetaVel) <= angle;
+  bool angleEqual = abs(expectedXVel - actual.xVel) <= vel;
+
+  if (velEqual && angleEqual) {
+    SUCCEED();
+  } else {
+    printf(
+        "Expected (thetaVel,xVel)=(%.5f,%.5f), "
+        "Actual (thetaVel,xVel)=(%.5f,%.5f)\n",
+        expectedThetaVel, expectedXVel, actual.thetaVel, actual.xVel);
+    FAIL();
+  }
```

Comment:
This could all be replaced with:
```
REQUIRE_THAT(actual.thetaVel, Catch::WithinAbs(expectedThetaVel, angle));
REQUIRE_THAT(actual.xVel, Catch::WithinAbs(expectedXVel, vel));
```
See: https://github.com/catchorg/Catch2/blob/devel/docs/comparing-floating-point-numbers.md